imageref.go

```diff
@@ -103,15 +98,59 @@ func (i *ImageRef) Export(outputFormat string) ([]byte, error) {
 	return result, nil
 }
 
-func (i *ImageRef) ExportWebp(params WebpExportParams) ([]byte, error) {
-	buf, _, err := i.ref.ExportWebp(&vips.WebpExportParams{
-		StripMetadata:   params.StripMetadata,
-		Quality:         params.Quality,
-		Lossless:        params.Lossless,
-		ReductionEffort: params.ReductionEffort,
-	})
+func (i *ImageRef) ExportWithParams(outputFormat string, params ExportParams) ([]byte, error) {
```

Comment:
i would rather export the `i.ref` instead proxying all the methods

Reply:
Maybe, but this method is not really a proxy